corehq/messaging/smsbackends/turn/models.py

```diff
@@ -100,7 +103,11 @@ def _send_fallback_message(self, msg):
     def get_all_templates(self):
         config = self.config
         client = TurnBusinessManagementClient(config.business_id, config.business_auth_token)
-        return client.message_templates.get_message_templates()
+        try:
+            # For bad config values, this can throw an exception
+            return client.message_templates.get_message_templates()
+        except JSONDecodeError:
```

Comment:
should this error be logged to sentry?

Reply:
I personally don't think so, because the user will get a notification on their platform saying their configuration is faulty and they should change it (this happens as the user enters a page).